parallel.js

```diff
@@ -51,7 +51,7 @@ var m = [60, 0, 10, 0],
 // };
 
 var colors = [
-  [0, 0 ,0],
+    [110,57,70]
 ];
```

Comment:
multi-line comments!

is this color not RGB? 128 128 128 did give me a white-like value but it should have been gray. 100 100 100 would be a darker gray by comparison but for some reason it showed up as the same. So, is this HSV instead of RGB?
What's a medium-gray that shows up in light and dark mode? @elliottregan